src/tools/base/BaseBrushTool.js

```diff
@@ -14,22 +14,8 @@ const { state, setters } = store.modules.brush;
  * @extends Tools.Base.BaseTool
  */
 class BaseBrushTool extends BaseTool {
-  constructor({
-    name,
-    strategies,
-    defaultStrategy,
-    configuration,
-    supportedInteractionTypes,
-    mixins,
-  }) {
-    super({
-      name,
-      strategies,
-      defaultStrategy,
-      configuration,
-      supportedInteractionTypes,
-      mixins,
-    });
+  constructor(...args) {
+    super(...args);
```

Comment:
You added these lines here, but not to `BaseAnnotationTool`?

Reply:
So, I can add it there too.. but as we were not doing anything else in the constructor off `BaseAnnotationTool`, it may be useless.

Reply:
I came back with the constructor.